client/performance-profiler/components/status-section/style.scss

```diff
@@ -9,7 +9,7 @@
 
 	.status-badge {
 		display: flex;
-		padding: 6px 0;
```

Comment:
Will this change affect the logged-out version?

Reply:
That component is unique to only logged in version